From 3e1324e713e8f3a588bb3ab8dc81cf06679406e8 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Tue, 9 Jul 2019 13:31:35 -0700
Subject: [PATCH 6/8] watchdog: digicolor_wdt: Remove unused variable in
 dc_wdt_probe

clang warns:

drivers/watchdog/digicolor_wdt.c:121:6: warning: unused variable 'ret'
[-Wunused-variable]
        int ret;
            ^
1 warning generated.

It's unused now, remove it.

Fixes: cdad26977e3f ("watchdog: digicolor_wdt: drop warning after registering device")
Link: https://github.com/ClangBuiltLinux/linux/issues/591
Reported-by: kernelci.org bot <bot@kernelci.org>
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 drivers/watchdog/digicolor_wdt.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/watchdog/digicolor_wdt.c b/drivers/watchdog/digicolor_wdt.c
index 33cda95bd238..073d37867f47 100644
--- a/drivers/watchdog/digicolor_wdt.c
+++ b/drivers/watchdog/digicolor_wdt.c
@@ -118,7 +118,6 @@ static int dc_wdt_probe(struct platform_device *pdev)
 {
 	struct device *dev = &pdev->dev;
 	struct dc_wdt *wdt;
-	int ret;
 
 	wdt = devm_kzalloc(dev, sizeof(struct dc_wdt), GFP_KERNEL);
 	if (!wdt)
-- 
2.22.0

